src/Phoenix.vue

```diff
@@ -62,6 +62,24 @@ export default {
 </script>
 
 <style>
+  .oc-app {
```

Comment:
that should be in the components - right?

Reply:
That should be in a Global Stylesheet tbh.

Reply:
isn't it the job of the component to take care of template, script and style? A component is an isolated unit -> all in one. This is also the power of the vue files from my understanding.

global is :hankey:

Reply:
global is shit.
but this is the only way i can controll the app behavior for all parts in focus of positon, size and layout. we can break it down to the core to make use of the components - which is actually part of a pattern library aswell imo...